betty/cropper/models.py

```diff
@@ -446,6 +450,12 @@ def get_absolute_url(self, ratio="original", width=600, format="jpg"):
             "extension": format
         })
 
+    def get_animated_url(self, format="gif"):
+        return reverse("betty.cropper.views.animated", kwargs={
+            "id": self.id_string,
+            "extension": format
```

Comment:
I realize this has been around for sometime now, but we should probably not use `format` since it is a built-in function